eng/pipelines/templates/steps/analyze.yml

```diff
@@ -30,7 +30,7 @@ steps:
 
       Copy-Item "$(Build.ArtifactStagingDirectory)/PackageInfo" -Destination $directoryToPublish -Recurse
     displayName: 'Create Go APIView Artifact'
-    condition: succeeded()
+    condition: and(succeeded(), ${{ ne(true, startsWith(parameters.ServiceDirectory, 'eng')) }})  # exclude changes to eng/
```

Comment:
Isn't IsSdkLibrary parameter already used for this? See condition on step before this.

Reply:
Ah, good catch. Maybe I can use that instead.